dts/bindings/sensor/maxim,max30101.yaml

```diff
@@ -1,8 +1,250 @@
+        setting this register. Set to 0 for no averaging.
+        0 = 1 sample (no averaging)
+        1 = 2 samples
+        2 = 4 samples
+        3 = 8 samples
+        4 = 16 samples
+        5 = 32 samples
+        6 = 32 samples
+        7 = 32 samples
+        Default: 0, same as after Power Reset
+    fifo-a-full:
+      type: int
+      required: false
+      default: 0
+      description: |
+        Set the trigger for the FIFO_A_FULL interrupt (e.g. if set to 2,
+        then the flag is set when the 30th word is written to the FIFO).
+        Range: 0-15, Default: 0, same as after Power Reset
+    fifo-rollover-en:
+      type: boolean
+      required: false
+      description: |
+        Controls the behavior of the FIFO when the FIFO becomes completely
+        filled with data. If set, the FIFO address rolls over to zero and the
+        FIFO continues to fill with new data. If not set, then the FIFO is
+        not updated until FIFO_DATA is read or the WRITE/READ pointer
+        positions are changed.
+    led-mode:
+      type: int
+      required: false
+      default: 2
+      enum:
+       - 2
+       - 3
+       - 7
+      description: |
+        Set the operation mode of the MAX30101.
+        2 = heart rate mode, only red led
+        3 = spo2 mode, red and ir led
+        7 = multi led mode, leds according configuration of slots
+        Default: 2, sufficient for heart rate recognition
+    led1-pa:
+      type: int
+      required: false
+      default: 0xff
+      description: |
+        Set the pulse amplitude to control the LED1 (red) current. The actual
+        measured LED current for each part can vary significantly due to the
+        trimming methodology.
+        0x00 = 0.0 mA
+        0x01 = 0.2 mA
+        0x02 = 0.4 mA
+        0x0f = 3.1 mA
+        0xff = 50.0 mA
+        Default: 0xff, full brightness for data quailty
+    led2-pa:
+      type: int
+      required: false
+      default: 0xff
+      description: |
+        Set the pulse amplitude to control the LED2 (IR) current. The actual
+        measured LED current for each part can vary significantly due to the
+        trimming methodology.
+        0x00 = 0.0 mA
+        0x01 = 0.2 mA
+        0x02 = 0.4 mA
+        0x0f = 3.1 mA
+        0xff = 50.0 mA
+        Default: 0xff, full brightness for data quailty
+    led3-pa:
+      type: int
+      required: false
+      default: 0xff
+      description: |
+        Set the pulse amplitude to control the LED3 (green) current. The actual
+        measured LED current for each part can vary significantly due to the
+        trimming methodology.
+        0x00 = 0.0 mA
+        0x01 = 0.2 mA
+        0x02 = 0.4 mA
+        0x0f = 3.1 mA
+        0xff = 50.0 mA
+        Default: 0xff, full brightness for data quailty
+    adc-rge:
+      type: int
+      required: false
+      default: 2
+      enum:
+       - 0
+       - 1
+       - 2
+       - 3
+       - 4
+       - 5
+       - 6
+       - 7
```

Comment:
Maybe you can add comments for each case here, instead of in the description.
BTW, why the description has only 4 cases?